Tests/MapboxMapsTests/Style/AttributionDialogTests.swift

```diff
@@ -0,0 +1,217 @@
+import XCTest
+@testable import MapboxMaps
+import Foundation
+import UIKit
+
+class AttributionDialogTests: XCTestCase {
+
+    var attributionDialogManager: AttributionDialogManager!
+    var mockDataSource: MockAttributionDataSource!
+    var mockDelegate: MockAttributionDialogManagerDelegate!
+
+    override func setUp() {
+        super.setUp()
+        mockDataSource = MockAttributionDataSource()
+        mockDelegate = MockAttributionDialogManagerDelegate()
+        attributionDialogManager = AttributionDialogManager(dataSource: mockDataSource, delegate: mockDelegate)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+
+        attributionDialogManager = nil
+        mockDataSource = nil
+        mockDelegate = nil
+    }
+
+    func testShowTelemetryDialogMetricsEnabled() throws {
+        let viewController = UIViewController()
+        let bundle = Bundle.mapboxMaps
+        let window = UIWindow()
+        window.makeKeyAndVisible()
+        window.rootViewController = viewController
```

Comment:
If this ordering results in a warning logged to the console, you may want to try setting rootViewController before calling makeKeyAndVisible